threads/apis/private.py

```diff
@@ -200,38 +200,53 @@ def get_recommended_users(self: PrivateThreadsApi, limit: int = 15, offset: Opti
 
         return response.json()
 
-    def get_user_followers(self: PrivateThreadsApi, id: int) -> dict:
+    def get_user_followers(self: PrivateThreadsApi, id: int, max_id: str | None = None) -> dict:
         """
         Get a user's followers.
 
         Arguments:
             id (int): a user's identifier.
+            max_id (str): offset.
 
         Returns:
             The list of user's followers inside a dict.
         """
-        response = requests.get(
-            url=f'{self.INSTAGRAM_API_URL}/friendships/{id}/followers/',
-            headers=self.headers,
-        )
+        if not max_id:
```

Comment:
Is there a need for this `if`?

It should work as expected if the argument will be passed as `None` — https://github.com/dmytrostriletskyi/threads-net/blob/main/threads/apis/private.py#L152

So no need to have a condition and decide whether to provide the argument. If it will be `None`, it will threat it as non-provided argument.

Reply:
Done